crt_portal/cts_forms/model_variables.py

```diff
@@ -19,14 +20,30 @@
     'Immigration/citizenship status (choosing this does not share your status)',
     'Religion',
     'Sex or gender identity (including gender stereotypes) or pregnancy',
-    'Sexual orientation',
     'Family, marriage, or parental status',
+    'Sexual orientation',
```

Comment:
I'm still seeing the old ordering here, what steps should I run to see the new ordering locally?